src/sbvr-api/sbvr-utils.coffee

```diff
@@ -540,14 +541,34 @@ exports.runRule = do ->
 		.nodeify(callback)
 
 exports.PinejsClient =
-	class PinejsClient extends PinejsClientCore(_, Promise)
-		_request: ({ method, url, body, tx, req, custom }) ->
+	class PinejsClient extends PinejsClientCore(_, Promise, mimemessage)
```

Comment:
Since we override `compileBatch` I don't think we actually need to pass in `mimemessage`?  Which would mean we can drop it as a dependency altogether